index.html

```diff
@@ -0,0 +1,368 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- start fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end fonts -->
+    <link rel="stylesheet" href="./static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <header class="nav" id="nav">
+      <div class="container">
+        <div class="header__container__parhead">
+          <div class="header__container__parhead__brand">
+            <a href="#">My Logo</a>
+          </div>
+
+          <nav class="header__container__parhead__bars">
+            <ul class="header__container__parhead__bars__responsive">
+              <li>
+                <img
+                  src="./static/imgs/Union@2x.png"
+                  alt="Union@2x"
+                  class="responsive header__container__parhead__bars__responsive__respo"
+                />
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <!-- end nav -->
+    <!-- start sectionone -->
```

Comment:
- [ ] Kindly make sure all the sections are grabbed inside a `main` tag that will be displayed `grid`